src/components/Field/FieldImagePreview.tsx

```diff
@@ -1,28 +1,53 @@
 import { ReactElement } from 'react';
-
+import { Button } from '@contentful/forma-36-react-components';
 import Imgix from 'react-imgix';
 
 import './FieldImagePreview.css';
 
 interface FieldImagePreviewProps {
   imagePath: string;
+  updateHeight: Function;
+  openDialog: Function;
+  clearSelection: Function;
 }
 
 export function FieldImagePreview({
   imagePath,
+  openDialog,
+  updateHeight,
+  clearSelection,
 }: FieldImagePreviewProps): ReactElement {
+  updateHeight(311);
```

Comment:
qq, why is `updateHeight(311)` vs some other value. Is that do with the preview image size?

Reply:
Good question, it's the value I found best matched the design comp.